packages/app/src/features/liquidity-flow/components/opacity-css-transition/opacity-css-transition.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { CSSTransition } from 'react-transition-group'
+import * as S from './styled'
+import { TRANSITION_CLASS_NAME } from './constants'
+
+interface Props {
+  inTransition: boolean
+  children?: React.ReactNode
+}
+
+export function OpacityCssTransition({ inTransition, children }: Props) {
+  return (
+    <CSSTransition in={inTransition} timeout={300} classNames={TRANSITION_CLASS_NAME}>
```

Comment:
Let's extract the magic number `300` to a constant.

Reply:
Added